src/code/z_elf_message.c

```diff
@@ -1,46 +1,60 @@
 #include "global.h"
+#include "z64elf_message.h"
 
-// TODO: use macros to define elf messages once the format is fully documented
 ElfMessage sChildSariaMsgs[] = {
-    { 0x66, 0x00, 0x03, 0x00 }, { 0x00, 0x37, 0x61, 0x00 }, { 0xE0, 0x00, 0x64, 0x00 }, { 0x00, 0x25, 0x62, 0x00 },
-    { 0x00, 0x37, 0x63, 0x00 }, { 0x00, 0x43, 0x65, 0x00 }, { 0x06, 0x30, 0x66, 0x66 }, { 0x06, 0x30, 0x66, 0x67 },
-    { 0x06, 0x30, 0x66, 0x68 }, { 0x06, 0x20, 0x67, 0x65 }, { 0x06, 0x30, 0x68, 0x69 }, { 0x06, 0x30, 0x68, 0x6A },
-    { 0xE0, 0x00, 0x69, 0x00 },
+/* 0000   0 */ ELF_MESSAGE_STRENGTH_UPG(BRANCHING, 3, false, 0), // TODO goron bracelet
+/* 0004   1 */ ELF_MESSAGE_FLAG(CHECK, 0x0161, false, 0x37), /* eventChkInf[3] & 0x80 */
```

Comment:
Having the full text ID in these commands seems very misleading since only the lowest byte will actually be used, especially in a modding scenario where people will probably tend to add entries in the 0x09XX which wouldn't work here.

So I think we should only have the lowest byte for every text ID (eg. 0x61 here)

Reply:
well, are we leaving Text IDs as raw numbers?